### PR TITLE
Fix/alert banner

### PR DESCRIPTION
**Pull request by DavitBakhutashvili**
#### Basic
- [ ] Used plop (`npm run plop`) to create a new component.
- [ ] PR has description.
- [ ] New component is functional and uses Hooks. 
- [ ] Component defines [`PropTypes`](https://reactjs.org/docs/typechecking-with-proptypes.html).
#### Style
- [ ] Styles are added to `NewComponent.modules.scss` file inside of the `NewComponent` folder.
- [ ] Component uses CSS Modules.
- [ ] Design is compatible with [L3vels Design System](https://design.l3vels.xyz/).
#### Storybook
- [ ] Stories were added to `/src/NewComponent/__stories__/NewComponent.stories.js` file.
- [ ] Stories include all flows of using the component.
#### Tests
- [ ] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

### CODE REVIEW

File: src/components/Toast/Toast.scss [hunk 1]
```diff
@@ -1,59 +1,72 @@
 @import "../../styles/global-css-settings";
 @import "../../styles/typography";
+@import "../../styles/typography/typography";
 @import "../../styles/themes";
 @import "../../styles/keyframes";
 
+$warning-background-color: linear-gradient(180deg, #fdfe53 0%, #eb9b3a 100%);
```
Comment (jemiks) [inline src/components/Toast/Toast.scss:7]: please don't use static colors use it from global

File: src/components/Toast/Toast.scss [hunk 2]
```diff
@@ -1,59 +1,72 @@
...
+  flex-direction: row;
+  justify-content: center;
+  border-radius: $toast-border-radius;
 
-  &-icon {
-    display: flex;
-    padding-left: $spacing-small;
-    padding-right: $spacing-small;
+  &-icon-small {
+    margin-left: 22.67px;
```
Comment (jemiks) [inline src/components/Toast/Toast.scss:27]: use for it variables which is created in a global CSS file

File: src/components/Toast/Toast.scss [hunk 3]
```diff
@@ -75,7 +88,7 @@
   }
 
   &_close-button {
-    margin-left: $spacing-small;
+    margin-right: 24px;
```
Comment (jemiks) [inline src/components/Toast/Toast.scss:91]: use varaibles

File: src/components/Toast/Toast.scss [hunk 4]
```diff
@@ -87,6 +100,33 @@
...
+  line-height: $font-line-height-small;
+}
+.l3-style-toast-paragraph {
+  @include typography-paragraph-small();
+  color: var(--color-transparent-black-05);
+  line-height: $font-line-height-small;
+}
+
+.l3-style-toast-artwork {
+  background: var (--color-transparent-white-02);
```
Comment (jemiks) [inline src/components/Toast/Toast.scss:120]: use varaibles

File: src/components/Toast/Toast.tsx [hunk 5]
```diff
@@ -100,36 +139,83 @@ const Toast: FC<ToastProps> & { types?: typeof ToastType; actionTypes?: typeof T
...
-            kind={Button.kinds.TERTIARY}
-            color={Button.colors.ON_PRIMARY_COLOR}
-            ariaLabel="close-toast"
+      <>
+        <div className={classNames} role="alert" aria-live="polite">
+          {avatar && (
+            <div className="l3-style-toast-avatar">
+              <Avatar
+                type={
+                  (avatarType === "img" && Avatar.types.IMG) ||
```
Comment (jemiks) [inline src/components/Toast/Toast.tsx:155]: type = {Avatar.type.avartype}

Comment (jemiks) [inline src/components/Toast/Toast.tsx:155]: remove this checks

File: src/components/Toast/Toast.tsx [hunk 6]
```diff
@@ -100,36 +139,83 @@ const Toast: FC<ToastProps> & { types?: typeof ToastType; actionTypes?: typeof T
...
+                  (avatarType === "icon" && Avatar.types.ICON) ||
+                  (avatarType === "text" && Avatar.types.TEXT)
+                }
+                size={
+                  (avatarSize === "small" && Avatar.sizes.SMALL) ||
```
Comment (jemiks) [inline src/components/Toast/Toast.tsx:160]: remove this check and send only one type which will be based on  avatarSize

File: src/components/Toast/Toast.tsx [hunk 7]
```diff
@@ -100,36 +139,83 @@ const Toast: FC<ToastProps> & { types?: typeof ToastType; actionTypes?: typeof T
...
+                withoutBorder={avatarBorder}
+                textClassName={avatarTextClassName}
+                backgroundColor={avatarBackgroundColor}
+                className={avatarClassName}
+                id={avatarId}
+                data-testid={avatarDataTestId}
+              />
+            </div>
+          )}
+          {(!artWork || !avatar) && iconSize === "small" ? (
```
Comment (jemiks) [inline src/components/Toast/Toast.tsx:181]: and medium? please create it dynamically  not use static checks

**Review by jemiks — CHANGES_REQUESTED** [review]

File: src/components/Toast/Toast.scss [hunk 8]
```diff
@@ -1,59 +1,166 @@
...
     }
     &-positive {
-      @include theme-prop(background-color, positive-color);
+      background: var(--color-gradient-green);
+      .l3-style-toast-label {
+        color: $black;
+        line-height: $font-line-height-small;
+        font-family: $toast-font-family;
+        font-style: $font-style-normal;
+        font-weight: $font-weight-bold;
```
Comment (jemiks) [inline src/components/Toast/Toast.scss:115]: remove this font styles and use typography

File: src/components/Toast/Toast.scss [hunk 9]
```diff
@@ -1,59 +1,166 @@
...
+        font-size: $label-font-size-medium;
+      }
+      .l3-style-toast-paragraph {
+        color: $black-5;
+        line-height: $font-line-height-small;
+        font-family: $toast-font-family;
+        font-style: $font-style-normal;
```
Comment (jemiks) [inline src/components/Toast/Toast.scss:122]: remove this font styles and use typography

File: src/components/Toast/Toast.tsx [hunk 10]
```diff
@@ -25,35 +26,76 @@ interface ToastProps extends L3ComponentProps {
...
   action: deprecatedAction,
   actions,
   children,
+  iconSize = "SMALL",
   closeable = true,
+  artWork,
+  avatar,
   onClose = NOOP,
-  className
+  className,
```
Comment (jemiks) [inline src/components/Toast/Toast.tsx:79]: I don't like this , there is so many avatar pros p

File: src/components/Toast/Toast.tsx [hunk 11]
```diff
@@ -100,35 +145,74 @@ const Toast: FC<ToastProps> & { types?: typeof ToastType; actionTypes?: typeof T
...
+                  className={avatarClassName}
+                  id={avatarId}
+                  data-testid={avatarDataTestId}
+                />
+              </div>
+            )}
+            {(!artWork || !avatar) && iconSize && <div className="l3-style-toast-icon-large">{iconElement}</div>}
+            {artWork && (
+              <div className="l3-style-toast-artwork">
+                {" "}
```
Comment (jemiks) [inline src/components/Toast/Toast.tsx:180]: remoe this empty spaces

**Review by jemiks — CHANGES_REQUESTED** [review]

File: src/components/Toast/Toast.tsx (hunk 7 above)
Comment (Chkhikvadze) [reply src/components/Toast/Toast.tsx]: @DavitBakhutashvili Also need check Tests

File: src/components/Toast/Toast.scss [hunk 12]
```diff
@@ -1,59 +1,132 @@
...
-  border-radius: $border-radius-small;
+  flex-direction: row;
+  justify-content: center;
+  border-radius: $toast-border-radius;
 
-  &-icon {
-    display: flex;
-    padding-left: $spacing-small;
-    padding-right: $spacing-small;
+  &--position {
```
Comment (jemiks) [inline src/components/Toast/Toast.scss:27]: this positioning will not work, the top left and the top right has the same styles try to fix it

**Review by jemiks — APPROVED** [review]

**Pull request merged**